client/src/components/Accordion/Accordion.scss

```diff
@@ -28,21 +28,22 @@
     line-height: $line-height-base;
     color: lighten($body-color, 30);
     text-align: center;
+    // equivalent of font-icon-up-open-big
     content: "7";
```

Comment:
The actual icon needs to change dynamically so we keep the `content` property in here but label what icon it is.